### DISCUSSION TITLE
Steam

### DISCUSSION THREAD

**Discussion by sgamesdev** [Leo-Corporation/Gavilya#450] [Q&A]
Is there anyway to get game's steam id in order to redirect to steam game page ?

**Comment by lpeyr** [comment]
Currently, you need to manually get the Steam ID for a game when creating a Steam Game. You can achieve this by copying the game ID in the URL of the game in the Steam Store.
If you want to open the Steam game page, you can use the following command: `steam://nav/games/details/<id>`, where `<id>` is the Steam Game ID.

**Reply by sgamesdev** [reply]
I see, I thought there was a way to get that  with the rawg api, because when your search something on rawg, you have a button {Steam} and it redirects you to the game's steam page.

**Reply by lpeyr** [reply]
Unfortunately, it doesn't seem that the RAWG API provided a store URL; it only returns the store information and URL, but not the URL to the game page itself.
